src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -42,7 +42,7 @@ final class JavaScriptImportPathCompiler implements AssetCompilerInterface
             (?:                            # Import statements (script captured)
                 import\s*
                     (?:
-                        (?:\*\s*as\s+\w+|\s+[\w\s{},*]+)
+                        (?:\*\s*as\s+\w+|\s+[\p{L}\w\s{},*]+)
```

Comment:
I don't know which encoding PCRE uses when the `u` modifier isn't set, but this doesn't work with e.g. character `é`.
I think the correct patch is to add the `u` modifier when the input is UTF-8 and contains non-ascii chars:
`self::IMPORT_PATTERN.(preg_match('{[^\\x00-\\x127]}u') ? 'u' : '')`
then this change here should be reverted